api/envoy/config/filter/network/http_connection_manager/v2/http_connection_manager.proto

```diff
@@ -520,6 +520,9 @@ message HttpConnectionManager {
   //
   // 3. Tracing decision (sampled, forced, etc) is set in 14th byte of the UUID.
   RequestIDExtension request_id_extension = 36;
+
+  // If true, envoy will retain keep-alive response headers from upstream for http1.1
+  bool retain_keepalive_response_header = 37;
```

Comment:
Assuming we move forward with this change, this should be reverted. The v2 APIs are dead.

Reply:
Thanks for confirming. Will keep it only in v3